app/models/asset.rb

```diff
@@ -185,12 +179,12 @@ def deleted?
     deleted_at.present?
   end
 
-protected
-
   def access_limited?
     access_limited.any? || access_limited_organisation_ids.any?
   end
 
+protected
```

Comment:
I wonder if it's worth writing unit tests for `access_limited?` now that it's a public function...? 🤔

Reply:
Nice that you kept this in its own commit 👍 but I'd add to the commit message to explain why it's being made public.